.github/workflows/testing.yml

```diff
@@ -8,14 +8,10 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: ["ubuntu-latest", "macos-latest", "windows-latest"]
-        python-version: ["3.8", "3.11"]
+        os: ["ubuntu-latest", "macos-13", "windows-latest"]
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
     steps:
-      - uses: s-weigand/setup-conda@v1
```

Comment:
`setup-conda` hasn't been updated in forever and is far less used than the official miniconda action